src/UserActivity.js

```diff
@@ -20,7 +20,7 @@ class UserActivity {
   minutesActive(activityDate, user) {
     let userActivity = this.data.filter((data) => data.date === activityDate);
     if (userActivity.length < 1) {
-      return "Sorry no data available for given date";
+      return 0;
```

Comment:
Nice, I was going to suggest this as a change in previous iterations but forgot, glad it's updated now!